ush/forecast_postdet.sh

```diff
@@ -19,8 +19,10 @@ FV3_postdet() {
     echo "Copying FV3 cold start files for 'RUN=${RUN}' at '${current_cycle}' from '${COMIN_ATMOS_INPUT}'"
     local fv3_file
     for fv3_file in ${file_list}; do
-      ${NCP} "${COMIN_ATMOS_INPUT}/${fv3_file}" "${DATA}/INPUT/${fv3_file}" \
-      || ( echo "FATAL ERROR: Unable to copy FV3 IC, ABORT!"; exit 1 )
+      if ! ${NCP} "${COMIN_ATMOS_INPUT}/${fv3_file}" "${DATA}/INPUT/${fv3_file}"; then
+        export err=1
+        err_chk "FATAL ERROR: Unable to copy FV3 IC, ABORT!"
+      fi
```

Comment:
I am sure this works, but instead, we should use `cpreq`.
```suggestion
      cpreq "${COMIN_ATMOS_INPUT}/${fv3_file}" "${DATA}/INPUT/${fv3_file}"
```
`cpreq` is the same as `cp` but the file is required, so a missing file will report an error.

Same comment for the ones below.